docs/getting-started/user-guides/testnet-voting.md

```diff
@@ -0,0 +1,183 @@
+
+The second step of this process is the actual voting. The previous 2016 block interval falls somewhere within a larger 5040 block interval and the network must wait for that larger 5040 block interval to end. Because of the different interval lengths, it *might* take up to an additional 5040 blocks before the voting window begins. After that, a static 5040 block interval expires while votes are cast and if 75% of the votes mined within that interval signal a 'yes' vote to the proposed changes, the changes are fully implemented after one additional block interval (to give any remaining nodes the time needed to update to avoid being forked off the blockchain). Below is a simplified chart to explain each block interval in the order they chronologically appear.
+
+Interval Description | Interval Type | Number of Blocks
+---------------------|-------------|---------------
+Minimum 75% of Votes must be newest version | Static | 2016
+Minimum 75% of Blocks must be newest version | Rolling | 100
+Interval after upgrade requirements met | Static | Up to 5040
+Actual voting interval - 75% of Votes must signal a "yes" to pass | Static | 5040
+Pre-implementation interval if vote passes | Static | 5040
+
+If a proposal fails to reach a 10% threshold of either 'no' or 'yes' votes, stakeholders will be able to vote on it again during the next block interval, until the threshold is met or the proposal expires. 
+
+Below are instructions to participate in the voting demo on the Testnet using a stakepool with Paymetheus and/or the command line applications `dcrd`, `dcrwallet`, and `dcrctl`. The command line guide uses configuration files to pass parameters to the application during launch. Alternatively, flags can be used when starting an application, but they will not be covered in this draft.
+
+## Paymetheus
+
+> Step 1: Download and Install Paymetheus
+
+If you haven't already updated your Decred binaries to v0.8.2, visit the [Installation Guide](/getting-started/install-guide.md) and follow the directions for the Windows Installer.
+
+> Step 2: Run Decred Testnet
+
+From the Start Menu, open `Decred Testnet`. This will launch `Paymetheus`, and a new Command Prompt window will open and run `dcrd.exe`. If this is the first time running the testnet daemon, it will take awhile to sync to the testnet blockchain. 
+
+In the `Paymetheus` window, you'll be greated by a "Connect to dcrd" dialog. Keep the defaults and press the continue button. The next view will have two buttons, "Create a new wallet" and "Restore wallet from seed." For this guide, it will be assumed you do not already have a seed you wish to restore.
+
+After clicking "Create a new wallet," you'll be greeted with information regarding your new wallet seed. Record your seed, put it in a safe place, and never share it with anyone. You will also need to re-enter it once you press the CONTINUE button.
```

Comment:
since this is testnet, backing up actually is not so necessary, and as a shortcut, you can just click through the confirm seed dialog without entering anything (this only works on testnet).  Not sure if you want to mention that last bit or not but it does make the testnet setup go a bit quicker.